docs/user-guide/registry/FAQ.md

```diff
@@ -1,94 +1,102 @@
 
-## 报错：https protocol is not supported
+## Exception：https protocol is not supported
 
-如下图所示：
+As shown in the following figure:
 
 ![https protocol is not supported](../../binary-docs/registry-faq-6.png)
 
-需要在插件配置（${agent_package_path}/agent/pluginPackage/service-registry/config/config.yaml）中，开启ssl（servicecomb.service.sslEnabled）。
+You need to enable ssl (servicecomb.service.sslEnabled) in the plugin configuration (${agent_package_path}/agent/pluginPackage/service-registry/config/config.yaml).
 
-## 报错：No such extension org.apache.dubbo.metadata.report.MetadataReportFactory by name sc
+## Exception：No such extension org.apache.dubbo.metadata.report.MetadataReportFactory by name sc
 
-如下图所示：
+As shown in the following figure:
 
 ![No such extension org.apache.dubbo.metadata.report.MetadataReportFactory by name sc](../../binary-docs/registry-faq-7.png)
 
-请检查dubbo应用的注册配置，protocol是否存在且不为sc，如下所示：
+Check the registration configuration of the dubbo application. Check whether protocol exists and is not sc.
 
-- 例如dubbo/provider.xml
+- Example for dubbo/provider.xml
 
 ```xml
 <dubbo:registry address="sc://127.0.0.1:30100" protocol="nacos"/>
 ```
 
-- 例如application.yml（或application.properties），以application.yml为例
+- For example, application.yml (or application.properties). The following uses application.yml as an example.
 ```yml
 dubbo:
   registry:
     protocol: nacos
     address: sc://127.0.0.1:30100
 ```
 
-如果protocol存在且不为sc，请把protocol的值设置成sc，或者删除protocol配置。
+If protocol exists and is not set to sc, set protocol to sc or delete the protocol configuration.
 
-## 报错：No registry config found or it's not a valid config
+## Exception：No registry config found or it's not a valid config
 
-如下图所示：
+As shown in the following figure:
 
 ![No registry config found or it's not a valid config](../../binary-docs/registry-faq-8.png)
 
-需要设置dubbo本身注册中心地址的配置，请参考[服务注册插件文档](./document.md#按需修改插件配置文件)中，关于**新开发**dubbo应用的说明。
+For details about how to set the address of the DUBBO registration center, see the description of newly developed DUBBO applications in the [service registration plugin](./document.md#Modify-the-plugin-configuration-file-on-demand) document.
 
-## 插件配置中，enableSpringRegister/enableDubboRegister与openMigration之间的关系是什么？
+## What is the relationship between Plugin configuration, enableSpringRegister/enableDubboRegister, and openMigration?
 
-enableSpringRegister/enableDubboRegister与openMigration之间的关系如下表所示：
+The following table describes the relationship between enableSpringRegister/enableDubboRegister and openMigration.
 
-|enableSpringRegister/enableDubboRegister|openMigration|作用|
+|enableSpringRegister/enableDubboRegister|openMigration|effect|
 |---|---|---|
-|true|true|开启spring cloud/dubbo迁移功能|
-|true|false|开启spring cloud/dubbo sc注册|
-|false|true|关闭注册插件|
-|false|false|关闭注册插件|
\ No newline at end of file
+|true|true|Enabling the Spring Cloud/Dubbo Migration Function|
+|true|false|Spring cloud/dubbo sc|
```

Comment:
maybe is "Enabling the Spring cloud/dubbo registration with the SC" ?

Reply:
Resolved